sys/androidmedia/gstamchybris.c

```diff
@@ -1186,11 +1186,6 @@ scan_codecs (GstPlugin * plugin)
 
       n_elems = media_codec_list_get_num_profile_levels (i, mime);
       GST_INFO ("Type '%s' has %d supported profile levels", mime, n_elems);
-      if (n_elems == 0) {
```

Comment:
VP8 does not have profiles, so this line was preventing its registration as an androidmedia plugin. No regressions exist as this simply was putting more restrictions on which codecs we considered valid. When the gstreamer registry is re-created, the previous codecs are created with the same properties as before.